forms-flow-api/tests/conftest.py

```diff
@@ -155,3 +157,59 @@ def get(self, key):
         return_value=mock_redis,
     ) as _mock:  # noqa
         yield mock_redis
+
+
+def get_form_request_payload():
+    """Return a form request payload object."""
+    return {
+        "formId": "1234",
+        "formName": "Sample form",
+        "processKey": "onestepapproval",
+        "processName": "One Step Approval",
+        "status": "active",
+        "comments": "test",
+        "tenant": 12,
+        "anonymous": False,
+        "formType": "form",
+        "parentFormId": "1234",
+    }
+
+
+@pytest.fixture
+def create_mapper():
+    """Create a mapper instance."""
+    mapper_data = FormProcessMapperSchema().load({**get_form_request_payload()})
+    response = FormProcessMapper.create_from_dict(
+        {**mapper_data, "created_by": "test", "tenant": None}
+    )
+    return FormProcessMapperSchema().dump(response)
+
+
+@pytest.fixture
+def create_mapper_anonymous():
+    """Create a mapper instance."""
```

Comment:
whether we need create_mapper_anonymous() can we reuse create_mapper_custom() for anonymous?

Reply:
we can do it later